src/share/doc/javatest/regtest/faq.md

```diff
@@ -699,6 +699,24 @@ if the default mode was not used.  For example,
     elapsed time (seconds): 0.141
 ````
 
+#### jtreg has trouble starting agents; what can I do?
+
+When running in agent mode, JTReg creates agent VMs as and when necessary.
+Agent VM creation involves launching a process and communicating with it over
+a socket. The initial handshake between the newly launched process and JTReg
+can sometimes timeout if the system is under heavy load. This then causes the
+agent creation to fail. By default, JTReg does not re-attempt creation of the
+agent VM and instead the failure is propagated as a test action failure and
+that action gets reported as a failed.
+
+This default behaviour can be overridden by passing the `--agent-attempts` option
+to `jtreg` command. This option takes an integer value which represents the number
+of attempts JTReg is allowed to make when attempting to get a agent for a test
```

Comment:
(minor) remove `JTReg is allowed` so that it reads "the number of attempts to make..."

Reply:
Done, updated the PR to remove that part.